src/hooks/use_toast.ts

```diff
@@ -0,0 +1,14 @@
+import {useToast as _useToast} from "@/components/shadcn-ui/use-toast";
+
+export const useToast = (error = false) => {
+  const {toast} = _useToast();
+
+  return (description: string, properties = {}) => {
```

Comment:
what is properties exactly ?
type your variables!

instead take conf in params
{ title, message, type: "err" | "success" }

default type is "success"
default title is "Uh oh..."
message is required